packages/webapp/src/components/MempoolToast/index.tsx

```diff
@@ -0,0 +1,35 @@
+import clsx from 'clsx';
+import { BigNumber } from 'ethers';
+import { formatEther } from 'ethers/lib/utils';
+import { BidTx, SettleTx } from '../../state/slices/mempool';
+import { shortAddress, shortENS } from '../../utils/addressAndENSDisplayUtils';
+import { useReverseENSLookUp } from '../../utils/ensLookup';
+import { buildEtherscanTxLink } from '../../utils/etherscan';
+import classes from './MempoolToast.module.css';
+
+const isBidTx = (c: any): c is BidTx => c.hasOwnProperty('value');
+
+const MempoolToast: React.FC<{ tx: SettleTx | BidTx }> = props => {
+  const { tx } = props
+
+  const ens = useReverseENSLookUp(tx.from)
+
+  const etherscanLink = buildEtherscanTxLink(tx.hash)
+  const ensOrAddress = ens ? shortENS(ens) : shortAddress(tx.from)
+  const value = isBidTx(tx) ? formatEther(BigNumber.from(tx.value)) : ''
+
+  const toastText = isBidTx(tx) ? `${ensOrAddress} just sent a new bid of ${value} Ξ` : `${ensOrAddress} is attempting to settle`
```

Comment:
How about these copy changes below? The first puts the bidding in the present tense like the settling, but also gives useful information about why it matters. The second changes technical language about "settling" to the real reason we care: the Noun on screen will be minted.
1) `${ensOrAddress} is sending a bid of ${value} Ξ. ${if (closeToAuctionEnd) "This will extend the auction by 5 minutes"}`
2) `${ensOrAddress} is attempting to mint this Noun`

Reply:
(I'm not sure how really long text toasts look. maybe trim down if necessary)

Reply:
Sadly, long texts won't look good, and it's hard to grasp long information, so we need to keep them "bite-sized". 

I have tried different copies, but the current ones fit the most. 

With `${ensOrAddress} is attempting to mint this Noun`, when settled outside of Fomo, it is rarely the case for particular noun reasons. In 80% of cases, it's an auction winner or bot that sends the tx, so the general `attempting to settle` will work the best I think. At least in the current realm.

Reply:
I agree, we don't know the intentions, but from a FOMO perspective 'settle' isn't really what's talked about. The point of FOMO is to mint a Noun (settling is how its done), so I *think* its clearer to say that someone else is minting vs settle.

Reply:
Agree. Changed that.